integr8ly/integreatly-operator
# INTLY-9315 - Refine critical SLO summary dashboard and fix non-instant alert state updates

# Description
- Adding 28 day limit on the queries to adhere to the SLA, especially in %SLO panel, previously the SLO percentage was non-indicative of the actual state, potentially miscalculating the SLA breach.
- Add instant selector to fetch alert states as they change, rather than previous approach of waiting 15 minutes.
- Add default dashboard refresh to 10 seconds, previously no refresh period was set and the dashboard would only update on manual refresh.
- Change default time range to now-5m (previously now-15m) meaning that states only got updated after 15 minutes from the actual change, no longer an issue due to `instant : true`, (now-5m) is still needed to update line graphs, the instant field on that query will break the graph.

## JIRA
https://issues.redhat.com/browse/INTLY-9315

## Type of change




## Checklist

## Verification
- Install this branch on your cluster.
- Run [E04](https://issues.redhat.com/browse/INTLY-9343) and ensure that the dashboard reflects the state of the alerts as seen in Prometheus UI
- Alerts should be updated immediately
- Critical SLO summary graph and % should grab metrics from the last 28 days
- Use the admin secret on the cluster to get an admin view of the graphs
